packages/tx-construction/src/tx-builder/OutputBuilder.ts

```diff
@@ -158,6 +153,11 @@ export class TxOutputBuilder implements OutputBuilder {
       }
     }
 
+    const outputValidation = toOutputValidationError(txOut, await this.#outputValidator.validateOutput(txOut));
```

Comment:
This was originally moved to have access to a resolved address but It's not needed now. 
883b14af431d6c40765f8350a71fa6995236fae6 should solve this

Reply:
@martynas when I run `yarn test:build:verify` I didn't see any error, but test failed after I pushed, so this code does need to be where it is in this line. 
However, we have a test for it, because on the tests I added we check whether the tx has a cardano.address, and if it doesn't it fails.  Which is the reason we moved it. 
I'll leave it like this, do you think it needs it further testing?

Reply:
If it's a `tsc` error otherwise then no need to do any further testing

Reply:
No, no it's not a `tsc` error, it's an actual data validation error, when there's no address, it fails.
<img width="1107" alt="Screenshot 2023-06-08 at 08 17 39" src="https://github.com/input-output-hk/cardano-js-sdk/assets/4052063/25e18122-6ab2-4bf9-8cb8-8a0fed4a3cdb">

Reply:
Ah I see, that's good too, seems to be covered by test already - nothing to do